note/index.js

```diff
@@ -0,0 +1 @@
+export * from './src';
```

Comment:
what is the reason to export stuff from src? I see you do it by default, but we are the DS for docs, by default there is no need to export anything, just `define` file is enough